components/suggest/src/store.rs

```diff
@@ -26,8 +23,8 @@ use crate::{
     provider::SuggestionProvider,
     rs::{
         SuggestAttachment, SuggestRecord, SuggestRecordId, SuggestRecordType,
-        SuggestRemoteSettingsClient, DEFAULT_RECORDS_TYPES, REMOTE_SETTINGS_COLLECTION,
-        SUGGESTIONS_PER_ATTACHMENT,
+        SuggestRemoteSettingsClient, SuggestRemoteSettingsRecord,
```

Comment:
Nit: those structs are getting harder to read, I feel like they can be simplified by dropping the prefixes or using some shorthanded replacements. All of them are defined in the `rs` mod of the `suggest` component, maybe both `Suggest` and `RemoteSettings` can be omitted from the struct IDs?

How about the following?
```rs
use rs::{Client, Record, RecordRequest};
// We can use aliases to disambiguate if needed

// or just follow what RS JS client's convention: use the prefix `Settings`
use rs::{SettingsClient, SettingsRecord, SettingsRecordRequest};
```
I don't have a strong preference, just wanted to hear your thoughts about this.

Reply:
I love the idea of shortening the names.  I'll open that as a follow-up PR.